sdk/eventhub/azure-eventhub/azure/eventhub/_eventprocessor/checkpoint_store.py

```diff
@@ -10,7 +10,7 @@
 class CheckpointStore(object):
     """CheckpointStore deals with the interaction with the chosen storage service.
 
-    It can list and claim partition ownerships; and list and save checkpoints.
+    It can list and claim partition ownerships; list and save checkpoints.
```

Comment:
Rather than removing the "and", I would replace it with "as well as"

Reply:
updated